ext/pdo_dblib/tests/batch_stmt_ins_exec.phpt

```diff
@@ -0,0 +1,61 @@
+--TEST--
+ PDO_DBLIB driver supports a batch of queries containing select, insert, update, exec statements
+ this test will fail when using the dblib driver that hasn't been patched
```

Comment:
This would be better described with an XFAIL section. Basically, you want to explain that this test is meant to identify an unresolved bug in FreeTDS.